assets/js/modules/search-console/datastore/report.js

```diff
@@ -88,37 +88,44 @@ const gatheringDataStore = createGatheringDataStore( 'search-console', {
 	dataAvailable:
 		global._googlesitekitModulesData?.[ 'data_available_search-console' ],
 	selectDataAvailability: createRegistrySelector( ( select ) => () => {
-		const rangeArgs = {
+		const url = select( CORE_SITE ).getCurrentEntityURL();
```

Comment:
I think the same comment applies as above, if this is recreating the report args used in `hasZeroData()` we should extract it to a shared utility function...

Reply:
The args used in `selectDataAvailability` and `hasZeroData` are the same, though. So I think it makes some sense to use a util function for this. Should I keep the change or revert?

Reply:
Ah, cool - well in that case, yes it's definitely a worthwhile change to make, let's keep it! :tada: